optimum/habana/transformers/integrations/gaudi_fused_sdpa_attention.py

```diff
@@ -18,13 +18,13 @@ def gaudi_fused_sdpa_attention_forward(
 ) -> tuple[torch.Tensor, None]:
     bsz, num_heads, tgt_len, head_dim = query.shape
 
+    softmax_mode = "fast" if os.getenv("FLASH_ATTENTION_FAST_SOFTMAX") == "1" else "None"
```

Comment:
since attention implementation is now separated from model we don't have to use env vars, can you explore if it's possible to use kwargs instead of env vars?

Reply:
Makes sense - I will have a look at it